### ISSUE TITLE
http.get never ends

### ISSUE THREAD

**Issue by millette** [olalonde/follow-redirects#7]
I'm using node v0.10.12 and I'm quite new to this.

This little program doesn't end using follow-redirects but it completes using the native http module:

```javascript

//var http = require('http');
var http = require('follow-redirects').http;

http.get('http://bit.ly/900913', function (res) {
  var all = '';
  res.on('data', function (chunk) {
    all = all + chunk;
    console.log(all.toString().length);
  });

  res.on('end', function (chunk) {
    console.log(all.toString().length);
  });
}).on('error', function (err) {
  console.error(err);
});

```

Am I missing something obvious? Or maybe follow-redirects needs to be updated to node 0.10? Any hint would be appreciated. I could slowly take a stab at updating the module if that's indeed the problem.

I wanted to take a stab at #6 but I have to figure this out first.

**Comment by jamestalmage** [comment]
Is this fixed?

**Comment by olalonde** [comment]
Sorry everyone. I think this is because I forgot to `npm publish` the module after @daviddenton fix was merged. Can anyone confirm it is working with a fresh `npm install follow-redirects` now?

**Comment by olalonde** [comment]
Hmmm seems it didn't work. Attempted to fix this again in 0.0.5. This is code I wrote quite a while ago so if this still doesn't work it would be nice if someone could attempt a PR :) The module is only ~100 LoC.

**Comment by jamestalmage** [comment]
I believe @olalonde took care of this in @1dd1218, published to npm as `0.0.5`. If anyone still has issues, do not hesitate to reopen.

**Comment by olalonde** [comment]
@jamestalmage thanks again for helping maintain this module :+1:
